Event/UserResolveEventInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Trikoder\Bundle\OAuth2Bundle\Event;
+
+use Symfony\Component\Security\Core\User\UserInterface;
+use Trikoder\Bundle\OAuth2Bundle\Model\Client;
+use Trikoder\Bundle\OAuth2Bundle\Model\Grant;
+
+/**
+ * Thi event occurs when the client requests a "password"
+ * grant type from the authorization server.
+ *
+ * You should set a valid user here if applicable.
+ */
+interface UserResolveEventInterface
```

Comment:
@Nyholm Any reason you didn't update `Tests/Integration/AuthorizationServerTest.php` & `Tests/Acceptance/TokenEndpointTest.php` to use the interface?

Also, `docs/password-grant-handling.md` should be updated as well.

Reply:
Nope, I just missed them.